code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/ProjectResourceManager.java

```diff
@@ -84,7 +86,7 @@ public Object listInternal(SchemaFactory schemaFactory, String type, Map<Object,
                     throw new ClientVisibleException(ResponseCodes.NOT_FOUND);
                 }
             }
-            policy.grantObjectAccess(project);
+            giveProjectAccess(project);
```

Comment:
We should probably pass policy object as an argument to giveProjectAccess call as we already retrieved it (line 70), and there is no need to call it for the second time (line 127)